app/guests+api.ts

```diff
@@ -1,9 +1,10 @@
-import { addGuest, getGuests, Guest } from '../database/guests';
+import { addGuest, getGuests } from '../database/guests';
 
-export function GET(request: Request): Response {
+export async function GET(request: Request): Promise<Response> {
```

Comment:
there's no way to type the response data of the Expo API response is there?

eg. like `NextResponse` in Next.js?

- https://github.com/vercel/next.js/issues/45943
- https://github.com/vercel/next.js/pull/47526

if not, we should probably:

1. see if it's possible to copy/paste a minimal implementation (eg. [this TypeScript Playground version](https://www.typescriptlang.org/play/?#code/MYewdgzgLgBAkgOQCoFEBKCCCAZAyjAXhlwE8BbAIxABsAKAcgEswoBTAJzAENqZ3WIAB3ARW9AJQAoSawAew9rGDUuECDARyoaAcMisAPACEQAExKEYAVzABrMCADuYAHwwtrMKfU6hI1jAA3pIwMADaiKgYOLgAugBcQSGhMCbmAPyJaSQANMkAvtKhoJBQ7FbAUCDstFQZWWYkcGCMsAA+MGBW1NQ5MMytib56os2tloH54kkpMBBWghy1jX0DUFLJoVAAFowQEcjoWHixE4WhhcnQXFCMwDAAVhDgBgBSz2DZLsvmie-g2VWLSgmRgw38Y3WiU0sm0un8bw+XxmKRK0D48P0Q0xoks4P0ADonuAfrl+sCpLN+FArJxOqxHBotPjRLR+H5CXUyeyRqxKTBCpc5ApYAAzGyVRjgGAAcRQSFo4mhzJxhkCXAowESAEYAEwAZnybkCMAA9KbZZ4OHcYFx2ABzKxkTywQSqUSmZKMUUwWgAWRu2wJ7C4XhAZEVMDcAAYCQBWaYmzYYml0mFwjmiIkfWjqzU6g1TM0WwCg5J0QO52OxqokuTBQFXWJUCslqbSwEzYSzWNmSYFTKxRYl6LJR-Qi+aYGWONX2LXGuSG-xm0L5NUxRLbtK5UhdYrlV3VUFixoK-arewbXbHc6WNYPV6ff7A8HQ6Zw5GY-HEzBk220yqmY9sSYC5hqWowHqhrTJOZYOJWs6JKGFh1lwwDAKwghsJ6FytqwqYdum3a9qB-aDsOo6yOOMGluWCE1raYAoQuaEYVhrCeoUQA)) to the Expo example repo (add to the Expo setup steps) - name could be something like `ExpoApiResponse`
2. open an Expo issue, if one doesn't already exist

Reply:
Generic type parameters for Expo API Routes Issue 
- https://github.com/expo/expo/issues/30521

Reply:
Added the types for the ExpoApiResponse in this commit
[98e83ec](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/98e83ec403f84e28ab454018bc92052f78a33fe5)

Reply:
`eslint-config-upleveled` updated:

- https://github.com/upleveled/eslint-config-upleveled/pull/447
- https://github.com/upleveled/eslint-config-upleveled/releases/tag/v8.10.0